_data/team.yml

```diff
@@ -0,0 +1,64 @@
+- name: Shelby Miller
+  title: Managing Partner
+  image: "shelby-miller.png"
+  classes: ""
+
+- name: Heather Spriggs
+  title: Controller
+  image: "heather-spriggs.png"
+  classes: ""
+
+- name: Machiko Yasuda
+  title: Software Engineer
+  image: "machiko-yasuda.png"
+  classes: ""
+
+- name: Scott Frazier
+  title: Managing Partner
+  image: "scott-frazier.png"
+  classes: ""
+
+- name: Laney Mangan
+  title: Associate Consultant, Customer Support Specialist
+  image: "laney-mangan.png"
+  classes: ""
+
+- name: Sarah Henry
+  title: UX Designer
+  image: "sarah-henry.png"
+  classes: "collapseMore collapse"
```

Comment:
In a future iteration I'd like to see us move presentation logic out of this file, and do something with e.g. Javascript to shuffle and show 5 images on page load.

This will make it easier for new team members to make an edit to this file to get themselves listed, and would make that page a little more fun/dynamic.